_studio/mfx_lib/encode_hw/h265/include/mfx_h265_encode_hw_utils.h

```diff
@@ -163,7 +163,7 @@ constexpr mfxU8    IDX_INVALID           = 0xff;
 constexpr mfxU8    HW_SURF_ALIGN_W       = 16;
 constexpr mfxU8    HW_SURF_ALIGN_H       = 16;
 
-constexpr mfxU8    MAX_SLICES            = 200; // WA for driver issue regarding CNL and older platforms
+constexpr mfxU16   MAX_SLICES            = 600;
```

Comment:
could you add a comment in code about what level limitation 600 slices is?

Reply:
Ok, I will do it

Reply:
Done